Pcap++/header/PcapFilter.h

```diff
@@ -484,113 +552,128 @@ namespace pcpp
 		 * Set the port
 		 * @param[in] port The port to create the filter with
 		 */
-		void setPort(uint16_t port) { portToString(port); }
+		void setPort(uint16_t port)
+		{
+			portToString(port);
+		}
 	};
 
-
-
 	/**
 	 * @class PortRangeFilter
-	 * A class for filtering TCP or UDP port ranges, meaning match only packets which port is within this range, for example: "src portrange 1000-2000"
-	 * will match only TCP or UDP traffic which source port is in the range of 1000 - 2000<BR>
-	 * For deeper understanding of the filter concept please refer to PcapFilter.h
+	 * A class for filtering TCP or UDP port ranges, meaning match only packets which port is within this range, for
+	 * example: "src portrange 1000-2000" will match only TCP or UDP traffic which source port is in the range of 1000 -
+	 * 2000<BR> For deeper understanding of the filter concept please refer to PcapFilter.h
```

Comment:
Maybe we can delete this `<BR>` and add a line break instead?

Reply:
Aren't the `<BR>` tags mostly for the html documentation?

Reply:
yes, I think we can remove it and add another line break

Reply:
Fixed a867a6c